apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
I don't need any of the elements of Dialog (which wraps BaseDialog). For example, here is my dialog
![image](https://cloud.githubusercontent.com/assets/1767466/17491358/2d98d59a-5d5c-11e6-99fa-b18c57f023e3.png)
and here is a `Dialog`
![image](https://cloud.githubusercontent.com/assets/1767466/17491379/3dc91628-5d5c-11e6-8fa0-b1d92bd7403a.png)

I don't have a title or icon, and my save/cancel buttons are different. Using BaseDialog got me the parts that I needed (hide/close, x in the corner, and a backdrop).